flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/source/reader/KafkaSourceReaderTest.java

```diff
@@ -83,7 +83,7 @@ public static void setup() throws Throwable {
         KafkaSourceTestEnv.setup();
         try (AdminClient adminClient = KafkaSourceTestEnv.getAdminClient()) {
             adminClient.createTopics(
-                    Collections.singleton(new NewTopic(TOPIC, NUM_SPLITS, (short) 1)));
+                    Collections.singleton(new NewTopic(TOPIC, KafkaSourceTestEnv.NUM_PARTITIONS, (short) 1)));
```

Comment:
Should we override the `getNumSplits()` method here to return `KafkaSourceTestEnv.NUM_PARTITIONS` as  well?

Reply:
Yes, sure, added in 1479815